src/doc/dev_manual/test_suite.rst

```diff
@@ -44,45 +43,51 @@ Here is an example of the contents of the generated ``run_visit_test_suite.sh``
+If any tests fail, ''all'' developers who updated the code from the last time all tests successfully passed will receive an email indicating what failed.
 In addition, failed results should be available on the web.  
 
 How regression testing works
 ----------------------------
 
-The workhorse script that manages the testing is ``visit_test_suite.py`` in 
-``src/test``. Tests can be run in a variety of ways called *modes*.
-For example, VisIt_'s nightly testing is run in ``serial``, ``parallel``
-and ``scalable,parallel`` modes. Each of these modes represents a fundamental and
-relatively global change in the way VisIt_ is doing business
-under the covers during its testing. For example, the difference
-between ``parallel`` and ``scalable,parallel`` modes is whether the scalable
-renderer is being used to render images. In the ``parallel`` mode,
-rendering is done in the viewer. In ``scalable,parallel`` mode, it
-is done, in parallel, on the engine and images from each processor
-are composited. Typically, the entire test suite is run in each
-mode specified by the regression test policy.
-
-There are a number
-of command-line options to the test suite. ``./run_visit_test_suite.sh -help``
-will give you details about these options. Until we are
-able to get re-baselined on the systems available outside of LLNL firewalls,
-options enabling some filtering of image differences will be very useful.
-Use of these options on platforms other than the currently adopted testing
-platform (pascal.llnl.gov) will facilitate filtering big
-differences (and probably real bugs that have been introduced)
-from differences due to platform where tests are run. See the section on
-filtering image differences.
+The workhorse script that manages the testing is ``visit_test_suite.py`` in ``src/test``.
+Tests can be run in a variety of ways called *modes*.
+For example, VisIt_'s nightly testing is run in ``serial``, ``parallel`` and ``scalable,parallel,icet`` modes.
+Each of these modes represents a fundamental and relatively global change in the way VisIt_ is doing business under the covers during its testing.
+For example, the difference between ``parallel`` and ``scalable,parallel,icet`` modes is whether the scalable renderer is being used to render images. In the ``parallel`` mode, rendering is done in the viewer.
+In ``scalable,parallel,icet`` mode, it is done, in parallel, on the engine and images from each processor are composited with `IceT <https://icet.sandia.gov>`_.
+Typically, the entire test suite is run in each mode specified by the regression test policy.
+
+The mode is specified with the ``-m`` command line option.
+For example, to run in ``scalable,parallel,icet`` mode use: ::
+
+    ./run_visit_test_suite.sh -n 1 -m "scalable,parallel,icet"
+
+Until we are able to get re-baselined on the systems available outside of LLNL firewalls, options enabling some filtering of image differences will be very useful.
```

Comment:
These options are described somewhere in the manual. Can we link to that section from here? (https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/test_suite.html?highlight=pixdiff#fuzzy-matching-thresholds)